lib/cdo/geocoder.rb

```diff
@@ -31,7 +31,7 @@ def self.find_potential_street_address(text)
     results = Geocoder.search(first_number_to_end)
     return nil if results.empty?
 
-    if results.first.types.include?('street_address')
+    if results.first.address
```

Comment:
Putting a comment here but I was trying to run this and it was crashing in `summarize` as a number of those attributes , such as `street_address`, don't exist for Mapbox. Can you take a look?